extensions/authentication-jwt/src/__tests__/fixtures/controllers/user.controller.ts

```diff
@@ -95,4 +106,68 @@ export class UserController {
   async whoAmI(): Promise<string> {
     return this.user[securityId];
   }
+  // Routes using refreshtoken
+  @post('/users/refresh/login', {
+    responses: {
+      '200': {
+        description: 'Token',
+        content: {
+          'application/json': {
+            schema: {
+              type: 'object',
+              properties: {
+                accessToken: {
+                  type: 'string',
+                },
+                refreshToken: {
+                  type: 'string',
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  })
+  async refreshLogin(
+    @requestBody(CredentialsRequestBody) credentials: Credentials,
+  ): Promise<TokenObject> {
+    // ensure the user exists, and the password is correct
+    const user = await this.userService.verifyCredentials(credentials);
+    // convert a User object into a UserProfile object (reduced set of properties)
+    const userProfile: UserProfile = this.userService.convertToUserProfile(
+      user,
+    );
+    const token = await this.jwtService.generateToken(userProfile);
+    const accessToken = await this.refreshService.generateToken(
+      userProfile,
+      token,
+    );
+    return accessToken;
+  }
+
+  @post('/refresh', {
+    responses: {
+      '200': {
+        description: 'Token',
+        content: {
+          'application/json': {
+            schema: {
+              type: 'object',
+              properties: {
+                refreshToken: {
+                  type: 'object',
```

Comment:
Better to have a more detailed schema with `properties` for `refreshToken`, like `RefreshGrantSchema` you defined in the other file.

Reply:
In this line as only access token is sent back, IMO it should be accessToken only in spite of refreshToken.